functions/weather.fish

```diff
@@ -49,15 +49,17 @@ function weather -d "Displays weather info"
   set wind_deg (echo $json | jq '.wind.deg')
 
   # convert m/s to km/h
-  set wind_speed_kmh ( math -s 1 $wind_speed \* 3.6 )
-  set  wind_gust_kmh ( math -s 1  $wind_gust \* 3.6 )
+  set wind_speed_kmh ( math -s 1 "$wind_speed * 3.6" )
+  if not test $wind_gust = null
+    set wind_gust_kmh ( math -s 1 "$wind_gust * 3.6" )
+  end
 
   # If we want miles per hour
-  #set wind_speed_mph ( math -s 1 $wind_speed \* 2.23694 )
+  #set wind_speed_mph ( math -s 1 "$wind_speed * 2.23694" )
```

Comment:
Also, please remove this from the diff, it's not part of the fix, besides being dead code – which is a subject of another commit.